services/core-web/common/models/documents/document.ts

```diff
@@ -0,0 +1,153 @@
+import { USER_ROLES } from "@mds/common";
+
+export enum FileOperations {
+  View = "Open in document viewer",
+  Download = "Download file",
+  Replace = "Replace file",
+  Archive = "Archive file",
+  Delete = "Delete",
+}
```

Comment:
Made sense to me to put this here, but would make sense to put it in the documents directory instead